lib/cybersource/models/response.rb

```diff
@@ -37,7 +37,7 @@ def self.cybersource_response_params(response)
             :params_cv_code => extract(response, 'cvCode'),
             :params_authorized_date_time => extract(response, 'authorizedDateTime'),
             :params_processor_response => extract(response, 'processorResponse'),
-            :params_reconciliation_id => extract(response, 'reconciliationID'),
+            :params_reconciliation_id => extract(response, 'reconciliationID') || extract(response, 'reconciliation_id'),
```

Comment:
the `reconciliationID` comes from Cybersource response, but in the case of transaction failures, there won't be this value. So use `reconciliation_id`, which is the value received in the request. see this line: https://github.com/killbill/killbill-cybersource-plugin/pull/20/files#diff-ac9d5b9b8d335656dab060b783053300R178